### ISSUE TITLE
Checkbox border color doesn't meet the 3:1 contrast ratio

### ISSUE THREAD

**Issue by Ma1h01** [tyler-technologies-oss/forge#996]
The default checkbox border color is `#9e9e9e`, which results in a 2.68:1 contrast ratio against the white background. Change it to `#949494` can fix the issue.

<img width="1910" height="1045" alt="Image" src="https://github.com/user-attachments/assets/dc235070-bd1b-4d33-95ee-3843bfdd68ed" />

<img width="510" height="770" alt="Image" src="https://github.com/user-attachments/assets/c51bb80d-cd48-46fa-ab01-4a032f1f0678" />
